src/plugins/intel_cpu/src/nodes/subgraph.h

```diff
@@ -77,22 +77,22 @@ class Snippet : public Node {
 
     class SnippetExecutor {
         public:
-            SnippetExecutor(SnippetAttrs attrs, bool is_dynamic, bool enforceBF16);
+            SnippetExecutor(SnippetAttrs attrs, bool is_dynamic, const GraphContext::CPtr& context);
```

Comment:
Do we really need the whole `GraphContext` in the executor? Maybe it's better to pass only one bool variable `enable_segfault_detector`?